### ISSUE TITLE
nu value is wrong for some of the regression bank knots

### ISSUE THREAD

**Issue by th-rtyf-re** [3-manifolds/knot_floer_homology#15]
In `python_src/HFK_data.json`, certain alternating knots have equal, negative `tau` and `nu`. This contradicts the definition https://github.com/3-manifolds/knot_floer_homology/blob/dcc4a757322946b470388c70b2237c3c67004d30/ComputeHFKv2/PyWrapper.cpp#L172
although I admit that I do not know what these values mean mathematically. Here are the culprits, with the bank and recalculated values, respectively:
```
    K14a14097: -2 != -1
    K14a14276: -2 != -1
    K14a15168: -1 != 0
    K14a4177: -1 != 0
    K15a50254: -3 != -2
```

**Comment by culler** [comment]
Is this related to the (now fixed) bug reported by Kyle Hayden, which only
affected the shortcut computation used for alternating knots?

- Marc

**Comment by th-rtyf-re** [comment]
I remember seeing something about nu values, yes, but I cannot remember where I saw it and I can't seem to find it anymore. I do think it is related to that bug.

**Comment by NathanDunfield** [comment]
Fixed by merging in #16.  Many thanks to @th-rtyf-re for finding this.
